### PR TITLE
XY side panel added

### PR DESCRIPTION
**Pull request by tmelorc**
I created tags for a xy side panel.
I hope I'm doing right.
I tested here and it works.

### CODE REVIEW

**Comment by sunderme** [comment]
I would prefer not having translations in the pull-request. They will be overwritten by transifex translations later ...

**Comment by tmelorc** [comment]
Hello @sunderme, thanks for letting me know.
Should I delete the translation files? I think not. But I am not going to update it further.
Sorry if I did some mistake. To test the side panel here I had to edit also some .cpp file.
Regards.

**Comment by sunderme** [comment]
> Hello @sunderme, thanks for letting me know.
> Should I delete the translation files? I think not. But I am not going to update it further.
> Sorry if I did some mistake. To test the side panel here I had to edit also some .cpp file.
> Regards.

no problem, I would just prefer the translation-files being reverted to master branch status, so that they don't show up as changed

File: src/texstudio.cpp [hunk 1]
```diff
@@ -7793,7 +7794,7 @@ void Texstudio::previewLatex()
 	}
 	if (!previewc.hasSelection()) {
 		// special handling for cusor in the middle of \[ or \]
-		if (c.previousChar() == '\\' && (c.nextChar() == '[' || c.nextChar() == ']')) {
+		if (c.previousChar() == '\\' and (c.nextChar() == '[' || c.nextChar() == ']')) {
```
Comment (sunderme) [inline src/texstudio.cpp]: I would prefer not to have c alternative operators in the source, please revert to '&&'

**Comment by tmelorc** [comment]
Oh, sorry. I can change the `and` but to be true, I have no idea what it is. I don't remember changing it. Strange.

**Comment by tmelorc** [comment]
I have just uploaded the cpp file and I believe now it is ok. Also, I changed the tag file. Please, update the files.
(git is tooo dificulty to me, sorry)
Regards

**Comment by sunderme** [comment]
at the moment, the pull-request has conflicts with master branch. Can you resolve this (rebase) ?

**Comment by sunderme** [comment]
it is not (directly) your fault.
You had Benito upload an older version and created a branch/pull-request from an obsolete version of the repository.
I suggest that you either read a little about git and how to merge the current master or negotiate with Benito to add newer version of your files (as delta to the current master branch !)

**Comment by tmelorc** [comment]
Well, I think it is better to close this pull request and some expert could take the files I created (the main one is the tags one and the icon) and help me to upload it.

**Comment by tmelorc** [comment]
I deleted the forked repository. I'll try to learn first how to fork the most updated one and then I can make a pull request.

**Pull request closed without merging**